src/Test.UnitTests.Sarif/TestData/Baseline/elfie-arriba.sarif

```diff
@@ -0,0 +1,237 @@
+                "region": {
+                  "startLine": 1,
+                  "startColumn": 1,
+                  "endLine": 1,
+                  "endColumn": 31,
+                  "charOffset": 0,
+                  "charLength": 31,
+                  "snippet": {
+                    "text": "-----BEGIN RSA PRIVATE KEY-----"
+                  }
+                }
+              }
+            }
+          ],
+          "partialFingerprints": {
+            "SecretHash/v1": "fdZwTjfxyQHIYf+BmyPXyLEOqdG4U2NLBuFNLckqc/s="
+          },
+          "properties": {
+            "matchState": "NotSuppressed"
+          }
+        },
+        {
+          "ruleId": "CSCAN0060/0",
+          "level": "error",
+          "message": {
+            "text": "Found PEM certificate file with private key."
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "file:///C:/Code/elfie-arriba/XForm/XForm.Web/node_modules/public-encrypt/test/test_rsa_privkey_encrypted.pem",
+                  "index": 3
+                },
+                "region": {
+                  "startLine": 1,
+                  "startColumn": 1,
+                  "endLine": 1,
+                  "endColumn": 31,
+                  "charOffset": 0,
+                  "charLength": 31,
+                  "snippet": {
+                    "text": "-----BEGIN RSA PRIVATE KEY-----"
+                  }
+                }
+              }
+            }
+          ],
+          "partialFingerprints": {
+            "SecretHash/v1": "fdZwTjfxyQHIYf+BmyPXyLEOqdG4U2NLBuFNLckqc/s="
+          },
+          "properties": {
+            "matchState": "NotSuppressed"
+          }
+        },
+        {
+          "ruleId": "CSCAN0020/0",
+          "level": "error",
+          "message": {
+            "text": "Found base64 encoded private key or certificate with private key in source file."
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "file:///C:/Code/elfie-arriba/XForm/XForm.Web/node_modules/spdy/test/fixtures.js",
+                  "index": 4
+                },
+                "region": {
+                  "startLine": 10,
+                  "startColumn": 8,
+                  "endLine": 10,
+                  "endColumn": 73,
+                  "charOffset": 160,
+                  "charLength": 65,
+                  "snippet": {
+                    "text": "'MIIEogIBAAKCAQEA1ARXSoyizYSnHDYickxX4x2UG/8uNWnQWKlWR97NAwRsspN6"
+                  }
+                }
+              }
+            }
+          ],
+          "partialFingerprints": {
+            "SecretHash/v1": "nHX0oMUmpq3IgCxodLjnV4F3ipb+Sj3SazMpzYQNZ9Y="
+          },
+          "properties": {
+            "matchState": "NotSuppressed"
+          }
+        }
+      ],
+      "automationDetails": {},
```

Comment:
>"automationDetails": {}, [](start = 6, length = 24)

The SDK (and all SARIF producers over which we have control) avoids emitting objects with no properties. How was this file generated?